src/main/java/com/findmyrecycling/fmrenterprise/service/FacilityServiceStub.java

```diff
@@ -1,12 +1,15 @@
 package com.findmyrecycling.fmrenterprise.service;
-
 import com.findmyrecycling.fmrenterprise.Decimal;
 import com.findmyrecycling.fmrenterprise.dto.Facility;
+import org.springframework.lang.Nullable;
 import org.springframework.stereotype.Component;
+import java.util.ArrayList;
+import java.util.List;
 
 @Component
 public class FacilityServiceStub implements IFacilityService{
     @java.lang.Override
+    @Nullable
```

Comment:
Instead of marking this nullable, fix the method so that it does what it says.

The risk is... you really don't want to return null.  So, once this is implemented, the @Nullable annotation shouldn't be there.  Its easy to miss that.